include/bluetooth/iso.h

```diff
@@ -337,49 +383,51 @@ struct bt_iso_server {
  */
 int bt_iso_server_register(struct bt_iso_server *server);
 
-/** @brief Bind ISO channels
+/** @brief Creates a CIG as a central
  *
- *  Bind ISO channels with existing ACL connections, Channel objects passed
- *  (over an address of it) shouldn't be instantiated in application as
- *  standalone.
+ *  This can called at any time, even before connecting to a remote device.
+ *  This must be called before any connected isochronous stream (CIS) channel
+ *  can be connected.
  *
- *  @param conns Array of ACL connection objects
- *  @param num_conns Number of connection objects
- *  @param chans Array of ISO Channel objects to be created
+ *  Once a CIG is created, the channels supplied in the @p param can be
+ *  connected using bt_iso_chan_connect.
+ *
+ *  @param[in]  param     The parameters used to create and enable the CIG.
+ *  @param[out] out_cig  Connected Isochronous Group object on success.
  *
  *  @return 0 in case of success or negative value in case of error.
  */
-int bt_iso_chan_bind(struct bt_conn **conns, uint8_t num_conns,
-		     struct bt_iso_chan **chans);
+int bt_iso_cig_create(const struct bt_iso_cig_create_param *param,
+		      struct bt_iso_cig **out_cig);
 
-/** @brief Unbind ISO channel
- *
- *  Unbind ISO channel from ACL connection, channel must be in BT_ISO_BOUND
- *  state.
+/** @brief Terminates a CIG as a central
  *
- *  Note: Channels which the ACL connection has been disconnected are unbind
- *  automatically.
+ *  All the CIS in the CIG shall be disconnected first.
  *
- *  @param chan Channel object.
+ *  @param cig    Pointer to the CIG structure.
  *
  *  @return 0 in case of success or negative value in case of error.
  */
-int bt_iso_chan_unbind(struct bt_iso_chan *chan);
+int bt_iso_cig_terminate(struct bt_iso_cig *cig);
 
-/** @brief Connect ISO channels
+/** @brief Connect ISO channels on ACL connections
+ *
+ *  Connect ISO channels. The ISO channels must have been initialized in a CIG
+ *  first by calling bt_iso_cig_create.
+ *
+ *  Once the connection is completed the channels' connected() callback will be
+ *  called. If the connection is rejected disconnected() callback is called
+ *  instead.
  *
- *  Connect ISO channels, once the connection is completed each channel
- *  connected() callback will be called. If the connection is rejected
- *  disconnected() callback is called instead.
- *  Channel object passed (over an address of it) as second parameter shouldn't
- *  be instantiated in application as standalone.
+ *  This function will also setup the ISO data path based on the @p path
+ *  parameter of the bt_iso_chan_io_qos for each channel.
  *
- *  @param chans Array of ISO channel objects
- *  @param num_chans Number of channel objects
+ *  @param param Pointer to a connect parameter with the ISO and ACL pointers.
+ *  @param count Number of connect parameters.
```

Comment:
Not clear to me.
Does the mean that the first parameter is the first in an array of bt_iso_connect_param?
Or is this the number of ISOs/ACLs in the param?

Reply:
It is an array, made that more clear.